src/sort_filter_funs.js

```diff
@@ -0,0 +1,26 @@
+
+export const SorterFun = (data,sortby) =>{
+    let prod = data
+    if(sortby ==="ltoh"){
```

Comment:
`ltoh` is definitely confusing. Rather have an understandable & maybe a longer name.